HelpSource/Classes/Server.schelp

```diff
@@ -922,6 +922,11 @@ Routine.run {
 )
 ::
 
+method:: quitSync
+Quit the Server and wait until the server process actually shuts down.
+argument:: condition
+an optional instance of link::Classes/Condition:: used for evaluating this.
```

Comment:
"used for evaluating this" is a bit unclear. Maybe: "to be  used within the synchronisation process"?

Reply:
I agree, but I copied it verbatim from [-bootSync description](https://github.com/supercollider/supercollider/pull/6738/files/e9922139f8dddf95cfba4e6315727b4b1978fca8#diff-7aa43aba358522fcf2dc1f56d8e54263d1554a55f5eec8186367ca523e6701e2L869). Should I update it only here or also in -bootSync?